pkg/Microsoft.NETCore.Targets/Microsoft.NETCore.Targets.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
 
   <PropertyGroup>
-    <PackageVersion>2.0.0</PackageVersion>
+    <PackageVersion>2.0.1</PackageVersion>
```

Comment:
For the packages we aren't building and shipping we shouldn't update the version number yet. Otherwise some packages might end up with a higher version of this package accidentally.